### ISSUE TITLE
Moving Pictures

### ISSUE THREAD

**Issue by heatherlyall** [heatherlyall/hide-show-responsively#1]
Hi,
I keep trying to get the pictures to move into a square instead of a straight line down for the index-400 screenshot, i've tried all the displays and none of them have worked. @thomasjbradley

**Comment by thomasjbradley** [comment]
Have you tried adding `flex-wrap: wrap`?

**Comment by heatherlyall** [comment]
Yea it's still going in a straight line @thomasjbradley

**Comment by thomasjbradley** [comment]
`display: flex` is on the wrong thing. It should be on `.dino-wrap`.

**Comment by heatherlyall** [comment]
now it's all in a horizontal line, is my flex-wrap in the correct place @thomasjbradley ?

**Comment by thomasjbradley** [comment]
Nope. `flex-wrap` always goes in the same place as `display: flex`.

**Comment by heatherlyall** [comment]
It's still having the same issue but now in a vertical line @thomasjbradley

**Comment by thomasjbradley** [comment]
Um. I don’t fully see the problem when I look at the website.
What’s happening?

**Comment by heatherlyall** [comment]
It's not flexing into a square with two pics at the top and two pics at the bottom like it's supposed to in one of the screenshots @thomasjbradley

**Comment by thomasjbradley** [comment]
Okay. Have you tried putting widths onto the `<div>` tags?

**Comment by heatherlyall** [comment]
Yes I've tried that, and even though it looks okay on the screen, when I put into markbot it shows me this @thomasjbradley 
<img width="387" alt="Screen Shot 2019-11-18 at 1 36 34 PM" src="https://user-images.githubusercontent.com/55193092/69079652-7c523580-0a08-11ea-949b-8d5e32eb8892.png">

**Comment by thomasjbradley** [comment]
Instead of using `display: flex` on `.dino` to make them visible again, use `display: block`.

**Comment by heatherlyall** [comment]
thank you so much markbot is happy with me now!! @thomasjbradley

**Comment by heatherlyall** [comment]
@thomasjbradley thanks so much!! last thing, markbot says that there is a parse error on 106 but doesn't there need to be a closing tag?

**Comment by thomasjbradley** [comment]
The 38 & 60 media queries are both missing their closing curly brackets.
